### ISSUE TITLE
Please update project / account url to the .com one

### ISSUE THREAD

**Issue by tscholze** [OpenLiveWriter/OpenLiveWriter#927]
The url of the project is pointing to the old .org one. 

**Screenshot**
<img width="693" alt="Bildschirmfoto 2020-09-28 um 11 05 23" src="https://user-images.githubusercontent.com/5583469/94412846-973e1c00-017a-11eb-8ccf-e8967db05a08.png">

**Comment by rkingett** [comment]
I honestly had no idea there was even a .com version. If this is newer, I second this suggestion.

**Comment by chrisjbreisch** [comment]
I don't believe it is newer. No updates on that site since 2017. I suspect they have just stopped paying their bills for their hostnames and that one hasn't expired yet.

**Comment by marknoble** [comment]
Hey @RobDolin & @willduff , saw you listed as the project contacts for this project.

https://dotnetfoundation.org/projects/open-live-writer

Seems like people are trying to fix things with this but they need some point people to coordinate the effort.

**Comment by dougrathbone** [comment]
Fix submitted in #1027 — updates hardcoded URLs from openlivewriter.org to openlivewriter.com with HTTPS.
